torch/onnx/utils.py

```diff
@@ -121,6 +121,81 @@ def _export(model, args, f, export_params=True, verbose=False, training=False,
     if isinstance(args, torch.autograd.Variable):
         args = (args, )
 
+    torch_out = None
```

Comment:
Good to put a comment at the top of `_export` updating the contract, since now it is not guaranteed to give you a `torch_out`.